draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -939,30 +939,17 @@ following conditions holds:
 
 ## "Same-site" and "cross-site" Requests  {#same-site-requests}
 
-Two origins, A and B, are considered same-site if the following algorithm
-returns true:
+Two origins are same-site if they satisfy the same-site criteria defined in
+{{FETCH}}. A request is "same-site" if the following criteria are true:
 
-1.  If A and B are both the same globally unique identifier, return true.
+1.  The request is not the result of a cross-site redirect. That is,
+    the origin of every url in the request's url list is same-site with the
+    request's current url's origin.
```

Comment:
Should "current url" be separately called out as a term that's defined in fetch?

Reply:
"current url" is defined in [here](https://github.com/httpwg/http-extensions/blame/master/draft-ietf-httpbis-rfc6265bis.md#L330), which I think is sufficient?

Reply:
Oops, I missed that we already had it there. Thanks!